slack-base/src/test/java/com/hubspot/slack/client/models/migration/MigrationExchangeParamsDeserializationTest.java

```diff
@@ -0,0 +1,64 @@
+package com.hubspot.slack.client.models.migration;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Set;
+
+import org.junit.Test;
+
+import com.hubspot.slack.client.jackson.ObjectMapperUtils;
+import com.hubspot.slack.client.models.JsonLoader;
+import com.hubspot.slack.client.models.response.migration.MigrationExchangeResponse;
+
+public class MigrationExchangeParamsDeserializationTest {
+
+  private static final String VALID_USER_OLD_ID = "U1X3XFPJL";
+  private static final String VALID_USER_MIGRATED_ID = "U01SZDTN10E";
+  private static final String INVALID_USER_ID = "INVALID_ID";
+
+  @Test
+  public void itDeserializesMigrationExchangeResponseWithValidAndinvalidUserIds() throws IOException {
+    MigrationExchangeResponse migrationExchangeResponse =
+        fetchAndDeserializeMigrationExchangeResponse("migration_exchange_response_with_valid_and_invalid_user_ids.json");
+    Map<String, String> userIdMap = migrationExchangeResponse.getUserIdMap();
+    Set<String> invalidUserIds = migrationExchangeResponse.getInvalidUserIds();
+    assertTrue(doesMainParamsPresent(migrationExchangeResponse));
+    assertTrue(userIdMap.containsKey(VALID_USER_OLD_ID) && userIdMap.containsValue(VALID_USER_MIGRATED_ID));
+    assertTrue(invalidUserIds.contains(INVALID_USER_ID));
```

Comment:
This looks a bit better to me:
```java
    assertMainResponseFields(migrationExchangeResponse);
    assertThat(userIdMap).containsKey(VALID_USER_OLD_ID);
    assertThat(userIdMap).containsValue(VALID_USER_MIGRATED_ID);
    assertThat(invalidUserIds).containsExactly(INVALID_USER_ID);
```
What do you think?

Reply:
Yes with assertThat it will return more precise errors.